packages/sdk/src/swap/types.ts

```diff
@@ -53,11 +53,15 @@ export interface SwapStatusRequest {
 }
 
 export interface CommonStatusFields extends ChainsAndAssets {
+  srcChain: Chain;
+  destChain: Chain;
+  srcAsset: Asset;
+  destAsset: Asset;
```

Comment:
this type extends `ChainsAndAssets` which should already contain this properties